python/pyspark/sql/catalog.py

```diff
@@ -108,13 +108,22 @@ def setCurrentCatalog(self, catalogName: str) -> None:
         ----------
         catalogName : str
             name of the catalog to set
+
+        Examples
+        --------
+        >>> spark.catalog.setCurrentCatalog("spark_catalog")
         """
         return self._jcatalog.setCurrentCatalog(catalogName)
 
     def listCatalogs(self) -> List[CatalogMetadata]:
         """Returns a list of catalogs in this session.
 
         .. versionadded:: 3.4.0
+
+        Returns
+        -------
+        list
+            A list of :class:`CatalogMetadata`.
```

Comment:
Seeing you add 
```
        >>> spark.catalog.listDatabases()
        [Database(name='default', catalog='spark_catalog', description='default database', ...
```

Why don't do the same for this API?

Reply:
Because it currently returns an empty catalog (that you're fixing at https://github.com/apache/spark/pull/37488)